pkg/cmd/admin/prune/images.go

```diff
@@ -27,9 +28,19 @@ import (
 )
 
 const (
-	imagesLongDesc = `%s %s - prunes images`
 	// PruneImagesRecommendedName is the recommended command name
 	PruneImagesRecommendedName = "images"
+
+	imagesLongDesc = `Prune images no longer needed by the system due to age and status.
+
+Only a user with a cluster role %s or higher who is logged-in will be able to actually delete
+the images.`
+
+	imagesExample = `# See, what the prune command would delete
```

Comment:
Explain what the flags will do. For examle, here all images older than 60 minutes and with more than 3 tags will be deleted.